app/labor/rate_limit_checker.rb

```diff
@@ -33,9 +33,9 @@ def limit_by_action(action)
   end
 
   def track_image_uploads
-    count = Rails.cache.read("#{@user.id}_image_upload").to_i
+    count = RedisRailsCache.read("#{@user.id}_image_upload").to_i
```

Comment:
While we are here making the ~world~ DEV a better place, let's change this to use [increment](https://apidock.com/rails/ActiveSupport/Cache/Store/increment) rather than implementing the logic ourselves. What do you think?